README.md

```diff
@@ -28,6 +28,45 @@ pip install git+https://github.com/huggingface/optimum-intel.git
 
 # Quick tour
 
+## Neural Compressor
+
+#### Dynamic quantization:
+
+Here is an example on how to apply dynamic quantization on a DistilBERT fine-tuned on the SQuAD1.0 dataset.
+Note that quantization is currently only supported for CPUs (only CPU backends are available), so we will not be utilizing GPUs / CUDA in this example.
+
+```python
+from transformers import AutoModelForQuestionAnswering
+from neural_compressor.config import PostTrainingQuantConfig
+from optimum.intel.neural_compressor import INCQuantizer
+
+model_name = "distilbert-base-cased-distilled-squad"
+model = AutoModelForQuestionAnswering.from_pretrained(model_name)
+# The directory where the quantized model will be saved
+save_dir = "quantized_model"
+# Load the quantization configuration detailing the quantization we wish to apply
+quantization_config = PostTrainingQuantConfig(approach="dynamic")
+quantizer = INCQuantizer.from_pretrained(model)
+# Apply dynamic quantization and save the resulting model in the ONNX format
+quantizer.quantize(quantization_config=quantization_config, save_directory=save_dir, save_onnx_model=True)
+```
+
+To load a quantized model hosted locally or on the 🤗 hub, you can do as follows :
+```python
+from optimum.intel.neural_compressor import INCQuantizedModelForSequenceClassification
+from optimum.onnxruntime import ORTModelForSequenceClassification
+
+# Load the ONNX model hosted locally
+onnx_model = ORTModelForSequenceClassification.from_pretrained(save_dir)
```

Comment:
1. should the ORTModelForSequenceClassification be ORT**Quantized**ModelForSequenceClassification
2. is asking user to explicitly invoke ORTModelXxx or INCQuantizedModelXxx a good way to load model? shall we make it transparent for user?

Reply:
`ORTModelForSequenceClassification` allows to load any ONNX model (quantized or not) so we shouldn't rename it to `ORTQuantizedModelForSequenceClassification` in my opinion. Something that we can think about is change `INCQuantizedModelXxx` to `INCModelXxx` to reduce the friction (even though we lose an important information, as `INCQuantizedModelXxx`  should be used for quantized model only)

Reply:
I don't think we should make it transparent as users would need `ORTModelForXxx` to load ONNX model and run inference with ONNX Runtime while `INCModelForXxx` is to load a PyTorch model quantized with INC, so both are for different case and requires different dependencies